frontend/src/komponenter/Oppsett.tsx

```diff
@@ -114,13 +126,7 @@ export default function Oppsett({
             </li>
           </ul>
         </nav>
-        <div className={`toaster ${visFeilmelding.display ? "active" : ""}`}>
-          {visFeilmelding.display && (
-            <Alertstripe onClick={() => dispatch(toasterSkjul())} type={visFeilmelding.type}>
-              <span>{feilmelding}</span>
-            </Alertstripe>
-          )}
-        </div>
+        {feilmeldingDisplay()}
```

Comment:
Motivasjonen for denne er litt ullen. Hvis du vil erstatte med en React komponent så bør den ganske enkelt være slik:

`<Feilmelding />`